### ISSUE TITLE
How to implement windows authentication ?

### ISSUE THREAD

**Issue by zhiv2u** [akveo/nebular#2225]
is there any way to set the httpclient  requestOptions.withCredentials = true while configuring the auth strategies?

**Comment by diogofalken** [comment]
Any information about this @sashaqred?

**Comment by sashaqred** [comment]
I'm not sure but it looks like it's not supported to do so with config. You can create your own auth strategy by extending the needed strategy. Something like `export class WithCredentialsAuthStrategy extends NbPasswordAuthStrategy {...}`

**Comment by diogofalken** [comment]
@sashaqred Thank you for your prompt response!

Creating a `Interceptor` that adds `withCredentials: true` to specified endpoints was the workaround I found.

Do you think it's worthwhile to create an issue to get this added as a config option? I noticed that you already have a `headers` optional config in version **9.0.0**, which may be something similar.

**Comment by sashaqred** [comment]
Yeah, the interceptor is also a good solution. I can't compare it with pros and cons, I just glanced at sources.

You can create a feature request, sounds like a good feature request.
